sphinx/templates/quickstart/root_doc.rst_t

```diff
@@ -3,19 +3,16 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to {{ project }}'s documentation!
-==========={{ project_underline }}=================
+{{ project }} documentation
+{{ project_underline }}==============
+
+Add your content using ``reStructuredText`` syntax. See the
+`reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/index.html>`_
+documentation for details.
+
 
 .. toctree::
    :maxdepth: {{ mastertocmaxdepth }}
    :caption: Contents:
 
 {{ mastertoctree }}
-
```

Comment:
I personally like the fact that we have indices and tables so I would not like to have it removed. In addition, it might be a bit hard for new users to remember those ones.

So instead, I'd suggest putting them in an RST comment *or* keep it altogether. Or maybe, only keep the search part and the others as comments.

Reply:
What benefit do they have for the user? IMHO

- ``:ref:`genindex` `` is more suited for printed documentation. For everything that's searchable (HTML, PDF) a search is usually more convenient and faster than scanning through an alphabetic index.
- ``:ref:`modindex` `` links to a missing page by default --> not a good user experience. And even for cases where you have added API documentation, you typically have only one or very few modules. There's not much to learn from this (e.g. https://requests.readthedocs.io/en/latest/py-modindex/).
- A link to the (empty) search page is not needed. You can start the search from the sidebar, which is more intuitive as every web page search works like this.

While it's great that this functionality exists, I suggest the applications are rather niche and not something we should clutter the default minimal template with.

Basically all documentation pages I know, including our own [sphinx](https://www.sphinx-doc.org/en/master/) and [alabaster](https://alabaster.readthedocs.io/en/latest/) pages do not contain "Indices and Tables".

Reply:
I think Bénédikt has a point in that many features of Sphinx can be hard to find, and the indicies and search from Sphinx are key features. This is a trade-off, though, and it may be that it is worth removing the whole section from the template.

A